.gitmodules

```diff
@@ -18,3 +18,6 @@
 	path = dumps/mongodb-sample-databases
 	url = https://github.com/mcampo2/mongodb-sample-databases.git
 	branch = master
+[submodule "tests/python-example"]
+	path = tests/python-example
+	url = https://github.com/FerretDB/python-example.git
```

Comment:
Please use the same format as submodule above – with a different name and with `branch` (that is important for dependabot)